consensus/ibft/ibft.go

```diff
@@ -847,6 +856,13 @@ func (i *Ibft) runAcceptState() { // start new round
 			return
 		}
 
+		if block.Number() != i.state.view.Sequence {
+			i.logger.Error("sequence not correct", "block", block.Number, "sequence", i.state.view.Sequence)
+			i.handleStateErr(errIncorrectBlockHeight)
+
+			return
+		}
```

Comment:
This check is already handled in the `blockchain` module:

https://github.com/0xPolygon/polygon-edge/blob/dad1ed048b7e0d7c54b9235b0663c1cc1eb6258c/blockchain/blockchain.go#L737-L746

Reply:
Hey, please don't talk about blockchain module 😂. I don't like removing required checks because other module does.

Reply:
Think of the nanoseconds we're wasting by doing this double checking 😢